lib/loadDefinedFile.js

```diff
@@ -2,8 +2,8 @@
 
 var yaml = require('js-yaml');
 var parseJson = require('parse-json');
-var requireFromString = require('require-from-string');
 var readFile = require('./readFile');
+var irequire = require('./require');
```

Comment:
I'd like to give this variable and the file more descriptive names. How about `importJs`?